rustfmt-core/rustfmt-lib/tests/source/space_around_attr_eq.rs

```diff
@@ -0,0 +1,6 @@
+// rustfmt-space_around_attr_eq: true
+
```

Comment:
Config option tests like this can go under the `configs/{option-name}/` directories within `source` and `target` (so `tests/source/configs/space_around_attr_eq/true.rs`
`tests/source/configs/space_around_attr_eq/false.rs`

https://github.com/rust-lang/rustfmt/tree/master/rustfmt-core/rustfmt-lib/tests/source/configs

Would be good to also have a test with the `false` value that shows the inverse too 👍

Reply:
Ah yes. I went down that path initially. I'll refactor the tests to use that pattern again.